components/server/src/workspace/image-source-provider.ts

```diff
@@ -36,25 +35,7 @@ export class ImageSourceProvider {
             let result: WorkspaceImageSource;
 
             const imgcfg = config.image;
-            if (ExternalImageConfigFile.is(imgcfg)) {
```

Comment:
@svenefftinge, this is actually created in https://github.com/gitpod-io/gitpod/pull/18316/files#diff-d26a92840b8fdcdeb54cc05627962891fbc4bbbe1f6872661f1e8a956f1d5600L109